renderer-macos/lib/components/EffectView.re

```diff
@@ -0,0 +1,43 @@
+open Brisk;
+
+module Material = BriskEffectView.Material;
+module BlendingMode = BriskEffectView.BlendingMode;
+
+type attr = [ Layout.style | Styles.viewStyle];
```

Comment:
Should it support stuff like `backgroundColor` and `border`?

Reply:
I'm not sure, I say we limit the scope of `EffectView` to handle only effects.